### ISSUE TITLE
Possible to have empty body in IResourceMethod?

### ISSUE THREAD

**Issue by TJMaria** [troyanskiy/rest-core#23]
Everything works perfectly except that when i want to POST an empty array as body i does not seem to be added as Request payload, only when the array is not empty.

Am i missing something?

ex:
`@ResourceAction({
        path: '/',
        method: ResourceRequestMethod.Post
    })
    postData: IResourceMethod<DataKey[], { name: string, version: string }>;`

**Comment by troyanskiy** [comment]
Hello.
Add a parameter `keepEmptyBody: true` to your `ResourceAction`

``` typescript
@ResourceAction({ 
  path: '/', 
  method: ResourceRequestMethod.Post,
  keepEmptyBody: true
}) 
postData: IResourceMethod<DataKey[], { name: string, version: string }>;
```

**Comment by TJMaria** [comment]
Hi, 

Thanks for the quick reply.
I did overlook that parameter, but it still does not seem to work for me.
I'm getting the exact same result.

**Comment by TJMaria** [comment]
Hi Raman, 

I made a pull request becaude the actionOption was never used in resource.ts

**Comment by troyanskiy** [comment]
I have released version 5.2.10.
Thanks for PR.

**Comment by TJMaria** [comment]
Wow. You're fast!
Thanks!
